lmdeploy/turbomind/generate_gemm_config.py

```diff
@@ -13,12 +17,80 @@ def get_llama_gemm():
     return bin_path
 
 
+def read_config(ini_path: str):
+    """read turbomind config from turbomind.
+
+    Args:
+        ini_path (str): the path of `config.ini` file in turbomind model
+    """
+    from configparser import ConfigParser
+    with open(ini_path, 'r') as f:
+        parser = ConfigParser()
+        parser.read_file(f)
+    section_name = 'llama'
+    _cfg = parser._sections[section_name]
+    cfg = TurbomindModelConfig.from_dict(_cfg)
+    return cfg['head_num'], cfg['size_per_head'], cfg['inter_size'], cfg[
```

Comment:
methods with unnamed tuple return is hard to maintain, similar to code smell [long parameter list](https://refactoring.guru/smells/long-parameter-list).
Would it be better to use a dataclass [parameter object](https://refactoring.guru/introduce-parameter-object)?